src/current/_includes/releases/whats-new-intro.md

```diff
@@ -58,6 +60,7 @@
 {% comment %}Is it skippable or LTS?{% endcomment %}
 
 {% if include.major_version.release_date != "N/A" %}
+  {% assign released = true %}
```

Comment:
The way we have determed whether a major-version line is released or not is the `release_date` in the `versions.csv`. The `releases.yml` has details about patches. Yes, a major version needs to have a `.0` release (but it could be withdrawn, see v23.1), but it's a lot to parse through all patches in all versions just to determine whether a major version is GA. Part of making it GA is setting its `release_date`, which determines all of its support dates. Before GA, patches have no support, and thus no support dates.

Reply:
But I should not have removed the check to be sure that `asst_supp_exp_date` is `N/A` and will put it back.